src/components/navBar/index.tsx

```diff
@@ -1,17 +1,50 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
+import React, { useState } from 'react';
```

Comment:
NIT: No need to import `React` it's imported by default in NextJs
https://github.com/vercel/next.js/issues/12964#issuecomment-629642157